packages/core/src/tools/timeUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { isNumber, round } from './utils'
+
+export type Duration = number & { d: 'Duration in ms' }
+export type ServerDuration = number & { s: 'Duration in ns' }
+export type TimeStamp = number & { t: 'Epoch time' }
+export type RelativeTime = number & { r: 'Time relative to navigation start' } & { d: 'Duration in ms' }
```

Comment:
Use different type keys to allow composition, we use RelativeTime as a Duration in ms